webapp/app/templates/lotse/form_person_a.html

```diff
@@ -4,52 +4,59 @@
 
 {% block form_content %}
 
-    <h2 class="mt-5 my-3">{{ _('form.lotse.person-header-allgemein') }}</h2>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_first_name, first_field=True if not render_info.form.errors) }}
-    </div>
-
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_last_name) }}
-    </div>
-
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_dob) }}
-    </div>
-
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_idnr) }}
-    </div>
-
-    <h2 class="mt-5 my-3">{{ _('form.lotse.person-header-addresse') }}</h2>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_street) }}
-        {{ macros.render_field(form.person_a_street_number, 2) }}
-    </div>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_street_number_ext) }}
-        {{ macros.render_field(form.person_a_address_ext) }}
-    </div>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_plz) }}
-        {{ macros.render_field(form.person_a_town, 6) }}
-    </div>
-
-    <h2 class="mt-5 my-3">{{ _('form.lotse.person-header-religion') }}</h2>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_religion) }}
-    </div>
-
-    <h2 class="mt-5 my-3">{{ _('form.lotse.person-header-beh') }}</h2>
-    <p class="mb-4">{{ _('form.lotse.person-header-beh-intro') }}</p>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_beh_grad, 4) }}
-    </div>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_blind, 4, d_block=True, hide_label=True) }}
-    </div>
-    <div class="form-row-center">
-        {{ macros.render_field(form.person_a_gehbeh, 4, d_block=True, hide_label=True) }}
+    <h2 class="mt-5 mb-0">{{ _('form.lotse.person-header-allgemein') }}</h2>
+    <div class="grouped-input-fields">
```

Comment:
Just out of naive curiosity: wouldn't `fieldset`s be a more natural / semantic way to group fields instead of `div`s?

Reply:
The reason I didn't do that is because we then have a `h2` as a legend of the fieldset. This was not recommended in the articles I found, but apparently those were outdated, as `hx` is an acceptable child of `legend` according to this documentation: https://html.spec.whatwg.org/multipage/form-elements.html#the-legend-element

Reply:
So I will change it to fieldset. Thanks 👍